source_modelling/sources.py

```diff
@@ -809,7 +810,7 @@ def __post_init__(self) -> None:
         This is essential for computing the shortest distances between faults.
         """
         # The trivial case where the number of planes is one should be ignored
-        if len(self.planes) == 1:
+        if len(self.planes) <= 1:
```

Comment:
Are they ever 0?

Reply:
sometimes, when the fault is constructed dynamically in the case of the NSHMDB